llarp/win32/version.rc.in

```diff
@@ -72,15 +72,15 @@ BEGIN
     BEGIN
         BLOCK "040904b0"
         BEGIN
-            VALUE "Comments", "includes relay/exit functionality, such code is highly experimental on non-Linux targets"
-            VALUE "CompanyName", "Loki Foundation"
-            VALUE "FileDescription", "LokiNET daemon for Microsoft� Windows� NT�"
-            VALUE "FileVersion", VERSION_STRING(LLARP_VERSION_TRIPLET, LLARP_RELEASE_MOTTO, VERSIONTAG)
-            VALUE "InternalName", "llarpd"
-            VALUE "LegalCopyright", "Copyright �2018-2020 Jeff Becker, Rick V for the Loki Foundation. All rights reserved. This software is provided under the terms of the zlib-libpng licence; see the file LICENSE for details."
-            VALUE "OriginalFilename", "llarpd.exe"
+            VALUE "Comments", "This comment has invoked its 5th ammendment constitutional right to remain silent"
```

Comment:
a note, the old value was out of date so i replaced it with a comment that says nothing.